### PR TITLE
Feature/COM-2598 Added ExprationDate and QuoteName Filter on Quote Grid 

### PR DESCRIPTION
**Pull request by hrishid**

### CODE REVIEW

File: scripts/modules/b2b-account/quotes.js [hunk 1]
```diff
@@ -21,12 +21,14 @@ define([
     CustomerModels, CartModels, B2BAccountModels, ProductModalViews,
     ProductPicker, ProductModels, WishlistModels, MozuGrid, MozuGridCollection,
     PagingViews, EditableView, QuoteModels) {
-
+        var NAMESTRING = "name cont ";
```
Comment (sagarraut0007) [inline scripts/modules/b2b-account/quotes.js]: Try if we could use 'const' for these variables if we are not going to reassign them anything later in the code

File: scripts/modules/b2b-account/quotes.js [hunk 2]
```diff
@@ -37,17 +39,74 @@ define([
         render: function () {
             var self = this;
             Backbone.MozuView.prototype.render.apply(this, arguments);
-            var collection = new QuotesGridCollectionModel({ autoload: true });
+            var collection = new QuotesGridCollectionModel({ autoload: true });            
+            $('[data-mz-action="applyfilter"]').on('change input', function(e) {
+                e.preventDefault();
+                FILTERSTRING = "";
+                var dateValue ="";
```
Comment (sagarraut0007) [inline scripts/modules/b2b-account/quotes.js]: rename 'dateValue' to like 'expirationDateValue'. make more meningful

File: scripts/modules/b2b-account/quotes.js [hunk 3]
```diff
@@ -37,17 +39,74 @@ define([
...
+        filterGrid: function (nameValue, dateValue, collection)
+        {
+            FILTERSTRING = "";
+             if(nameValue!=="")
+             {
+                 nameValue = NAMESTRING + nameValue;
+                 FILTERSTRING = nameValue;
+                 if(dateValue!=="")
+                 {
+                     dateValue = DateString + dateValue+"T00:00:00z";
```
Comment (sagarraut0007) [inline scripts/modules/b2b-account/quotes.js]: Make this hardcoded timestamp as a 'const'

File: scripts/modules/b2b-account/quotes.js [hunk 4]
```diff
@@ -21,12 +21,14 @@ define([
     CustomerModels, CartModels, B2BAccountModels, ProductModalViews,
     ProductPicker, ProductModels, WishlistModels, MozuGrid, MozuGridCollection,
     PagingViews, EditableView, QuoteModels) {
-
+        var NAMESTRING = "name cont ";
+        var DateString = "expirationdate ge ";
```
Comment (sagarraut0007) [inline scripts/modules/b2b-account/quotes.js]: rename 'DateString' to maybe something like 'expirationDateFilter'

File: stylesheets/modules/b2b-account/orders.less [hunk 5]
```diff
@@ -6,7 +6,24 @@
           margin: 0 @gutter/4 0 0;
       }
    }
-   
+
+   #searchName
```
Comment (sagarraut0007) [inline stylesheets/modules/b2b-account/orders.less]: Move all of these styles to quotes.less file. since these are not related to orders

**Review by sagarraut0007 — CHANGES_REQUESTED** [review]

**Review by rakesh-dontula — COMMENTED** [review]
@hrishid are all the code review comments taken care of on this PR? It helps if you can reply to every comment made by reviewer to know if all the requested changes are taken care.

**Review by rakesh-dontula — APPROVED** [review]

File: scripts/modules/b2b-account/quotes.js (hunk 1 above)
Comment (hrishid) [reply scripts/modules/b2b-account/quotes.js]: const is es6 and is not supported

File: scripts/modules/b2b-account/quotes.js (hunk 2 above)
Comment (hrishid) [reply scripts/modules/b2b-account/quotes.js]: done

File: scripts/modules/b2b-account/quotes.js (hunk 4 above)
Comment (hrishid) [reply scripts/modules/b2b-account/quotes.js]: done

File: stylesheets/modules/b2b-account/orders.less (hunk 5 above)
Comment (hrishid) [reply stylesheets/modules/b2b-account/orders.less]: done

File: scripts/modules/b2b-account/quotes.js (hunk 3 above)
Comment (hrishid) [reply scripts/modules/b2b-account/quotes.js]: const not supported but seperated it in a variable and used.

**Comment by ghost** [comment]
This PR was mistakenly merged into another feature branch. This story was not yet done. This merged commit has BEEN DELETED and this PR has been superseded by #273.
